docs/spec.md

```diff
@@ -787,21 +789,21 @@ defined as follows:
 * `flattened_ids(replica_groups)`
   if `channel_id > 0 and use_global_device_ids = true`.
 
-Afterwards, within each `process_group`:
+Afterwards, within each `process_group` and for each i in [0, size(operands)]:
```

Comment:
Similarly, remove the iteration on `i`

```
* `results...@process[results_index] = exec(schedule)` for some binary tree
  `schedule` where:
  * `exec(node)` = `computation(exec(node.left), exec(node.right))`.
  * `exec(leaf)` = `leaf.value`.
* `schedule` is an implementation-defined binary tree whose in-order
  traversal is `to_destination_type(operands...@process_group...[results_index],
  type(func_inputs(computation)[0]))`.

```

Reply:
updated. instead of `results_index` used `result_index`